robottelo/host_helpers/satellite_mixins.py

```diff
@@ -39,6 +39,18 @@ def enable_puppet_satellite(self):
         assert 'Success!' in result.stdout
         return self
 
+    def enable_multicv_setting(self):
+        cfg_file = 'upstream_only_settings.rb'
+        cfg_path = self.execute(f'find /usr/share/gems/gems/ -name {cfg_file}').stdout.strip()
+        assert cfg_file in cfg_path, 'Config file not found'
+        self.execute(
+            f'sed -i "s/allow_multiple_content_views/#allow_multiple_content_views/g" {cfg_path}'
```

Comment:
function is about enable but `sed` actually disables the setting ...

Reply:
ah! 
UpstreamOnlySettings - so you need to excluded that so it gets applied to Stream/Downstream

Reply:
Yeah it looks odd, but here we actually remove the setting from `UpstreamOnlySettings`, making it available downstream.